app/models/profile_rule.rb

```diff
@@ -2,10 +2,10 @@
 
 # Join table to be able to have a has-many-belongs-to-many relation between
 # Profile and Rule
-class ProfileRule < ApplicationRecord
+class ProfileRule < ProfileLink
   belongs_to :profile
```

Comment:
This line is common for both `Rule` and `RuleGroup` so please move it to the class above. Shared stuff should not be repeated if possible.